README.md

```diff
@@ -1,280 +1,70 @@
-```
-      - name: upstream.state.up
-        source: func:StringEquals(upstream.server.state,up)
-        type: gauge
-        label: state up
-        description: Server is up
-```
+After cloning the repo, executing `build.sh` will build the packages for multiple Linux distributions.
 
-- TimeSince - Returns time in ms passed since event was accured. Simple calculation of now() - time_contained_in_metric_param
+### Setup
+The packages can be installed using OS specific package manager like dpkg, yum, etc. Once installed a new App can be 
+setup by running `setup-spm` command. For example, to setup monitoring for a JVM application in standalone mode, add
 
+```bash
+-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=3000 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false
 ```
-      - name: uptime.ms
-        source: func:TimeSince(broker.start.time.ms)
-        type: double_gauge
-        unit: ms
-```
-
-- TrimUnit - Trims the specified unit string from the end of value of the metricName and returns Long. E.g. TrimUnit(Value,ms) - trims `ms` from  the end of value of metric name `Value`
+VM arguments to the Java startup process. Then you can setup new app by running the following command:
 
-```
-      - name: indexing.commits.auto.time.max
-        source: func:TrimUnit(autocommit maxTime,ms)
-        type: gauge
-        label: autocommit max time
-        description: autocommit max time
-        unit: ms
+```bash
+sudo bash /opt/spm/bin/setup-spm  \
+    --app-token <app-token>   \
+    --app-type jvm  \
+    --agent-type standalone \
+    --jmx-params '-Dspm.remote.jmx.url=localhost:3000'
 ```
 
-### Service type specific built-in functions:
-- com.sematext.spm.client.hadoop.CalculateNumNodes - hadoop specific. The only param is metric name mapped to hadoop jmx attribute LiveNodes.
+`<app-token>` - App Token is should point to Sematext App Token if you are sending metrics to Sematext. For other Influx endpoints
+you can specify a hexa-decimal value with format `xxxxxxxx--xxxx-xxxx-xxxx-xxxxxxxxxxxx`. e.g. `d0add28a-0a0f-46b2-9e1e-4928db5200e7`
 
-```
-      - name: nn.nodes.live
-        source: func:com.sematext.spm.client.hadoop.CalculateNumNodes(nodes.live)
-        type: long_gauge
-        label: live nodes
-```
+Visit [Sematext Documentation](https://sematext.com/docs/monitoring/spm-client/) for more info on how to setup and 
+configure agent to ship metrics.
 
-- com.sematext.spm.client.solr.CalculateCoreNumFiles - solr specific, calculates number of files for some solr core (can't use universal CountFiles function due to Solr internal specifics)
+By default the agent sends the collected metrics to Sematext. You can configure different Influx compatible destination
```

Comment:
"to Sematext" -> "to Sematext Cloud"?

Reply:
I think "Sematext" is OK, too.  We use that a lot elsewhere.

different => a different